ManagedNativeInterop/ManagedNativeInterop/ManagedNativeInterop.csproj

```diff
@@ -88,7 +91,7 @@
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
     <RuntimeIdentifier>win10-arm64</RuntimeIdentifier>
-    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```

Comment:
nit: windows arm64 does not support coreclr, so we need to set UseDotNetNativeToolchain to true here